keps/1282-pods-ready-requeue-strategy/README.md

```diff
@@ -75,25 +105,69 @@ Consider including folks who also work outside the SIG or subproject.
 
 ### API Changes
 
-Add an additional field to the Kueue ConfigMap to allow administrators to specify what timestamp to consider during queue sorting (under the pre-existing waitForPodsReady block).
+#### KueueConfig
+
+Add an additional field to the KueueConfig to allow administrators to specify what timestamp to consider during queue sorting (under the pre-existing waitForPodsReady block).
 
 Possible settings:
 
 * `Eviction` (Back of queue)
 * `Creation` (Front of queue)
 
-```yaml
-kind: ConfigMap
-metadata:
-  name: kueue-manager-config
-  namespace: kueue-system
-data:
-  controller_manager_config.yaml: |
-    apiVersion: config.kueue.x-k8s.io/v1beta1
-    kind: Configuration
-    # ...
-    waitForPodsReady:
-      requeuingTimestamp: Creation | Eviction # <-- New field
+```go
+type WaitForPodsReady struct {
+	...
+	// requeuingStrategy defines the strategy for requeuing a Workload
+	// +optional
+	RequeuingStrategy *RequeuingStrategy `json:"requeuingStrategy,omitempty"`
```

Comment:
I think that we can break this API for now since we haven't yet released this API.